app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -3,5 +3,144 @@ $brand-primary: #cc0000;
 // turn buttons square
 $border-radius-base: 0;
 
+
 @import "bootstrap-sprockets";
-@import "bootstrap"
+@import "bootstrap";
+
+body {
+  padding-bottom: 80px;
+}
+
+.navbar {
+  height: 60px;
+  background-color: #0E426C;
```

Comment:
The color `#0E426C` repeats everywhere. Assign it to a SASS variable at the top of the file and replace instances of `#0E426C` with your variable.

Something like 👇 
```css
$base-background-color: #0E426C
```
Then, you can use `$base-background-color` in place of `#0E426C`.